frontend/src/views/groupmember/AddGroupMember.vue

```diff
@@ -0,0 +1,313 @@
+<template>
+  <div id="addGroupMember" v-if="addMode">
+    <form @submit.prevent="submitForm">
+      <AppRow>
+        <AppCol class="col3">
+          <AppInput :e-model="v$.groupNumber" id="groupNumber" label="Group Number" type="text" v-model.trim="groupNumber"/>
+        </AppCol>
+          <AppCol class="col3">
+          <AppDateInput :e-model="v$.coverageEffectiveDate" id="dp-input-coverageEffectiveDate" label="Coverage Effective Date" tooltip tooltipText="Date always defaults to first day of the month"
```

Comment:
The id shouldn't have "dp-input-" as a prefix. This is something that the datepicker component will assign.